src/components/Breadcrumbs.js

```diff
@@ -49,7 +49,8 @@ function Breadcrumbs({ history, rootPathLength }) {
             {crumbs.map((crumb, index) => {
                 if (index < crumbs.length - 1) {
                     return (
-                        <Fragment>
+                        // eslint-disable-next-line react/no-array-index-key
+                        <Fragment key={index}>
```

Comment:
That comment is just a directive to stop the linter complaining about it. In general using indexes as keys is bad but if you're certain the list never needs to be sorted its ok. Since these are Breadcrumbs order is important and as such sorting isn't likely.. so didn't feel too bad doing it here. There are a few places where I've used it a bit more dangerously though... For example in the ReportTable - if we ever implement a sortable table I'll need to rethink map keys.

Reply:
I think I'm scared hiding linting stuff where we can avoid it but that's fair enough! Yeah saw report table too but I remember us saying ideally it needs rewritten at some point anyway